packages/ripple-keypairs/.eslintrc.js

```diff
@@ -56,5 +56,8 @@ module.exports = {
     'eslint-comments/require-description': 'off',
     'no-shadow': 'off',
     'multiline-comment-style': 'off',
+    '@typescript-eslint/no-require-imports': 'off',
+    'import/no-commonjs': 'off',
+    '@typescript-eslint/no-var-requires': 'off',
```

Comment:
It looks like this is correct syntax to avoid this error:
`import foo = require('foo');`

Reply:
I removed this rule as it seems I fixed it already in the code and this was left over from my testing

Reply:
Is it possible to add a comment to this line explaining that this is a rule which should not be turned on? (Otherwise if someone follows the TODO above to start turning on linting rules they could get confused and think that this is something they need to fix across the repo)